client/src/store.ts

```diff
@@ -112,6 +124,8 @@ const initialState = {
 	description: '',
 	selectedAction: 'Migrate',
 	selectedScript: '',
+	currentProject: '',
```

Comment:
Nice, I put this under selectedProject for my branch but we can roll with yours.